diff --git a/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh b/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
index b13a70be37b..36a56965cc1 100755
--- a/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
+++ b/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
@@ -115,6 +115,7 @@ case "${TENSORFLOW_TARGET}" in
     ;;
 esac
 
+BUILD_NUM_JOBS=$(grep -c ^processor /proc/cpuinfo)
 cmake --build . --verbose -j ${BUILD_NUM_JOBS} -t _pywrap_tensorflow_interpreter_wrapper
 cd "${BUILD_DIR}"
 
